### ISSUE TITLE
Exception thrown after page reload.

### ISSUE THREAD

**Issue by rmartonik** [WellCommerce/WellCommerce#39]
In admin panel, from time to time, reload using reload button in web browser throws following exception:

LogicException in RequestHelper.php line 92: Cannot set session attributes without valid session.

**Comment by hacfi** [comment]
@rmartonik You have to provide more information: With OS, webserver + versions are you running on? What do you use as the session handler? http://symfony.com/doc/2.8/cookbook/session/sessions_directory.html

Also check the logs in `app/logs` or `var/logs` for additional information about the error. Or you webserver/php logs.

**Comment by adampiotrowski** [comment]
@hacfi It happens since we've upgraded Symfony dependencies to 3.x. Every time the cache is cleared or invalidated, without warming it up (cache:warmup), first request will always throw an exception. Consecutive requests are fine so that must be something with classes.php or dumped container imho. Will do an investigation ;)

**Comment by hacfi** [comment]
Cloning the repo now..will have a quick look!

**Comment by hacfi** [comment]
FYI:
```
  - Installing wellcommerce/composer-plugin (dev-master 9a72afd)
    Cloning 9a72afddecd9b86136afffc1060e2e8eb0c94e24

The "wellcommerce/composer-plugin" plugin requires composer-plugin-api 1.0.0, this *WILL* break in the future and it should be fixed ASAP (require ^1.0 for example).
```

Not sure why specifying a specific version seems to be an issue but the `^1.0` version constraint doesn’t sound bad to me.

**Comment by adampiotrowski** [comment]
@hacfi Thanks! Issue created: https://github.com/WellCommerce/WellCommerce/issues/41
